src/dialog/llm/abstract_llm.py

```diff
@@ -79,9 +79,11 @@ def process(self, input: str):
         """
         processed_input = self.preprocess(input)
         self.generate_prompt(processed_input)
+
         if len(self.relevant_contents) == 0 and \
-            self.config.get("prompt").get("fallback_not_found_relevant_contents"):
-            return {"text": self.config.get("prompt").get("fallback_not_found_relevant_contents")}
+            self.config.get("prompt").get("fallback"):
+            return {"text": self.config.get("prompt").get("fallback")}
+
```

Comment:
@avelino as you can see here, if we have an if statement that doesn't use the LLM inside the `process(self, input)` function, we are good